### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by powerjg**

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -1,6 +1,82 @@
 # gem5art tasks package
 
-This package contains a [Celery](http://www.celeryproject.org/) worker and tasks for running gem5.
+This package contains two parallel task libraries for running gem5 experiments.
+he actual gem5 experiment can be executed with the help of [Python multiprocessing support](https://docs.python.org/3/library/multiprocessing.html), [Celery](http://www.celeryproject.org/) or even without using any job manager (a job can be directly launched by calling `run()` function of gem5Run object).
 This package implicitly depends on the gem5art run package.
 
-See [gem5art](https://github.com/darchr/gem5art) repo for more details.
+Please cite the the [gem5art paper](https://arch.cs.ucdavis.edu/papers/2021-3-28-gem5art) when using the gem5art packages.
```
Comment (aakahlow) [inline tasks/README.md]: extra "the"!

File: tasks/README.md [hunk 2]
```diff
@@ -1,6 +1,82 @@
 # gem5art tasks package
 
-This package contains a [Celery](http://www.celeryproject.org/) worker and tasks for running gem5.
+This package contains two parallel task libraries for running gem5 experiments.
```
Comment (aakahlow) [inline tasks/README.md:3]: So, just to make sure: we are going to keep this documentation both on the website and in the gem5art source as well?

**Review by aakahlow — APPROVED** [review]

File: tasks/README.md (hunk 2 above)
Comment (powerjg) [reply tasks/README.md:3]: That's my thought. We'll just have the */README.md in the source and all of the docs/ on the gem5 website.

File: tasks/README.md (hunk 1 above)
Comment (powerjg) [reply tasks/README.md]: Good find! Though, you missed it in the run and artifact readmes :laughing: I just copy-pasted this line each time.

**Pull request merged**